README.md

```diff
@@ -1,2 +1,197 @@
 # planship-nuxt
-Planship Nuxt module
+
+Welcome to `@planship/nuxt`, a [Nuxt module](https://nuxt.com/modules) that makes [Planship](https://planship.io) intergration with [Nuxt](https://nuxt.com) apps a breeze. This module is based on the [`@planship/vue`](https://github.com/planship/planship-vue) plugin and [`@planship/fetch`](https://github.com/planship/planship-js/tree/master/packages/fetch) library.
```

Comment:
intergration -> integration

Consider rewording to include general functionality that the module (Planship) exposes:
> Welcome to `@planship/nuxt`, a [Nuxt module](https://nuxt.com/modules) that enables entitlements, metering, packaging, and customer/subscription management in your [Nuxt](https://nuxt.com) app powered by [Planship](https://planship.io). This module is based on the [`@planship/vue`](https://github.com/planship/planship-vue) plugin and [`@planship/fetch`](https://github.com/planship/planship-js/tree/master/packages/fetch) library.